.gitmodules

```diff
@@ -0,0 +1,5 @@
+[submodule "common/cryptolib"]
+	path = common/security/cryptolib
+	url = git@github.com:tiiuae/cryptolib.git
```

Comment:
I'd pull in submodules as http to avoid issues with initialisation after cloning
```suggestion
	url = https://github.com/tiiuae/cryptolib.git
```